### PR TITLE
Update prelogin urls

### PR DESCRIPTION
**Pull request by biyeun**
move prelogin site to be at root. this will require further updates to apache conf files on deploy

@orangejenny (@TylerSheffels fyi)

### CODE REVIEW

File: corehq/apps/hqwebapp/urls.py [hunk 1]
```diff
@@ -3,8 +3,9 @@
 
 urlpatterns = patterns(
     'corehq.apps.hqwebapp.views',
+    (r'^$', 'redirect_to_default'),
     url(r'^homepage/$', 'redirect_to_default', name='homepage'),
-    url(r'^home/$', 'landing_page', name='landing_page'),
+    url(r'^default_landing/$', 'landing_page', name='landing_page'),
```
Comment (millerdev) [inline corehq/apps/hqwebapp/urls.py]: What will happen to people who go to `/home/` once this is in effect?

Personally I think `http://commcarehq.org/home/` is prettier than `http://commcarehq.org/default_landing/`. Will anyone ever see this URL?

File: urls.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
     from localsettings import PRELOGIN_APP_URLS
 except ImportError:
     PRELOGIN_APP_URLS = (
-        (r'^public/', include('corehq.apps.prelogin.urls')),
+        (r'', include('corehq.apps.prelogin.urls')),
```
Comment (millerdev) [inline urls.py]: Does this match anything or is the empty string treated the same as `'^$'`?

File: corehq/apps/hqwebapp/urls.py (hunk 1 above)
Comment (biyeun) [inline corehq/apps/hqwebapp/urls.py]: no one will see this url unless they are running hq locally. Right now `/home` is being hijacked by apache, so no one even sees that.

Comment (biyeun) [inline corehq/apps/hqwebapp/urls.py]: if you have `default_landing` you are not running the prelogin site / haven't pulled the repo

File: urls.py (hunk 2 above)
Comment (biyeun) [inline urls.py]: it's treated the same as `^`. see the hqwebapp `urlpattern`

**Pull request merged**